src/dispatcher/mod.rs

```diff
@@ -168,6 +190,7 @@ impl Dispatcher {
                 .arg("up")
                 .arg("-n")
                 .arg(&name)
+                .arg(&accept_prompt.unwrap())
```

Comment:
Should avoid unwrap as much as possible. Unless you can guarantee it will never panic.